compose.integration.yaml

```diff
@@ -0,0 +1,20 @@
+# Docker compose file for integration environment
+# Run with: docker compose up
+
+services:
+  webapp:
+    # Set platform to linux/amd64 when building on ARM based system to push to ECR
+    # platform: linux/amd64
+    build:
+      context: .
```

Comment:
I put this in slack, but I'll put in here too, since it probably makes more sense here! I have `target: test` or `target: development` in compose.yaml (dev) and compose.test.yaml so that it runs only the specified stages from the Dockerfile. What happens if you don't specify like here? Do we need to tell it not to run the test or development stages, or does it somehow know?

Reply:
I will also put the relevant information here!

From the docker documentation, the final stage (final) is the default target for building.

https://docs.docker.com/get-started/docker-concepts/building-images/multi-stage-builds/#:~:text=In%20your%20multi%2Dstage%20Dockerfile,the%20last%20stage%20by%20default.

Reply:
Thank you!